pyscriptjs/src/components/base.ts

```diff
@@ -158,6 +156,12 @@ export class BaseEvalElement extends HTMLElement {
                 this.outputElement.style.display = 'block';
             }
 
+            if (is_async) {
+              await pyodide.runPythonAsync(`output_manager.revert()`);
+            } else {
+              await pyodide.runPython(`output_manager.revert()`);
+	    }
```

Comment:
Hi, maybe this line gets the linter error.

```suggestion
            }
```